Formula/f/fuego.rb

```diff
@@ -2,10 +2,16 @@ class Fuego < Formula
   desc "Collection of C++ libraries for the game of Go"
   homepage "https://fuego.sourceforge.net/"
   url "https://svn.code.sf.net/p/fuego/code/trunk", revision: "1981"
-  version "1.1.SVN"
-  revision 7
+  version "1.1"
+  license any_of: ["GPL-3.0-only", "LGPL-3.0-only"]
+  revision 8
```

Comment:
```suggestion
  revision 8
  version_scheme 1
```

We will have to bump the `version_scheme` to switch from `1.1.SVN` to `1.1`, as the former is seen as newer from the standpoint of `Version` comparison.